src/pdf.rs

```diff
@@ -387,10 +261,11 @@ mod test {
 
         let custom_writer = CustomWriter(0);
 
-        let surface = Writer::new(20., 20., custom_writer);
+        let surface = PdfSurface::for_stream(20., 20., custom_writer);
         surface.set_size(100., 100.);
         draw(&surface);
-        let custom_writer = surface.finish();
+        surface.finish();
+        let custom_writer = surface.take_output_stream().unwrap().downcast::<CustomWriter>().unwrap();
```

Comment:
Minor: Please use `expect` instead of `unwrap`, makes errors easier to spot when they occur (even more in tests!).